pyews/core.py

```diff
@@ -61,5 +78,5 @@ def invoke(self, soap_body):
             __LOGGER__.info("A Timeout Error occurred attempting to connect to {ep}:".format(ep=endpoint) + repr(errt))
         except requests.exceptions.RequestException as err:
             __LOGGER__.info("An Unknown Error occurred attempting to connect to {ep}:".format(ep=endpoint) + repr(err))
-        __LOGGER__.warning('Unable to parse response from {current}'.format(current=self.__class__.__name__))
```

Comment:
It doesn't look like appropriate log message, because when BS can't parse 'ResponseCode', 'ErrorCode' or anything else it's just return None object. Furthermore, even if BS raised exception during parsing, structure like this wouldn't catch the error rather then:
```
try:
...
except requests.exceptions.Exception:
    ...
except Exception:
    ...  # now it can handle the error
```
In https://github.com/swimlane/pyews/issues/15 me and author of one met an 'AttributeError' for instance.